README.md

```diff
@@ -6,21 +6,10 @@
 [![Dependency Status](https://img.shields.io/david/cgiffard/node-simplecrawler.svg)](https://david-dm.org/cgiffard/node-simplecrawler)
 [![devDependency Status](https://img.shields.io/david/dev/cgiffard/node-simplecrawler.svg)](https://david-dm.org/cgiffard/node-simplecrawler#info=devDependencies)
 
-Simplecrawler is designed to provide the most basic possible API for crawling
-websites, while being as flexible and robust as possible. I wrote simplecrawler
-to archive, analyse, and search some very large websites. It has happily chewed
-through 50,000 pages and written tens of gigabytes to disk without issue.
-
-#### Example (simple mode)
```

Comment:
The reason I removed it was because of its brevity compared to how extensive simplecrawler's set of options are, and the slightly more extensive code examples are very near if one scrolls down just a bit. Still, maybe you're right - it might be a good way to get an instant picture of the module, even if it's very brief